packages/did/src/Did.chain.ts

```diff
@@ -173,15 +157,13 @@ export async function generateCreateTx({
   alg,
   didIdentifier,
   keys = {},
-  endpointData,
 }: IDidCreationOptions & KeystoreSigningOptions): Promise<
   SubmittableExtrinsic
 > {
   const blockchain = await BlockchainApiConnection.getConnectionOrConnect()
   const encoded = encodeDidCreationOperation(blockchain.api.registry, {
```

Comment:
Q: I might be confused, but I don't see a submitter added to the creation TX. Didn't we want to define the submitter specifically for the creation operation?
And why do we have the submitter for the `generateDidAuthenticatedTx` calls, where the deposit is not changed AFAIK?